docs/run_test/platform.rst

```diff
@@ -9,7 +9,77 @@ Run tests on different platforms
 Run on Azure
 ------------
 
-Follow the same procedure in :doc:`Getting started with Azure <quick_run>`.
+VM can be deployed on Azure using images from vhd, shared image
+gallery or marketplace. If multiple types are specified, the first
+non-empty type is picked in the following order :
+vhd, shared image gallery and marketplace.
+
+Running using vhd
+^^^^^^^^^^^^^^^^^
+To run using vhd, add the following to runbook :
+
+.. code:: yaml
+   platform:
+   - type: azure
+      ...
+      requirement:
+         ...
+         azure:
+            ...
+            vhd: "<VHD URL>"
+
+The `<VHD URL>` can either be a SAS url or a blob url. If it is
+a SAS url, the image is copied it to a location in the subscription
```

Comment:
the image is copied into?

Reply:
It's a storage account. I will add the info.